app/services/metrics_service.rb

```diff
@@ -13,9 +13,19 @@ def fetch_time_series(base_path:, date_range:)
     api.client.get_json(url).to_hash
   end
 
+  def content_summary(date_range:, organisation:)
+    api_with_long_timeout.content_summary(from: date_range.from, to: date_range.to, organisation: organisation)
+  end
+
 private
 
   def api
     @api ||= GdsApi::ContentDataApi.new
   end
+
+  def api_with_long_timeout
+    api.tap do |client|
+      client.options[:timeout] = 30
```

Comment:
Not sure if `30 seconds` of timeout on the client side
matches the timeout defined in Nginx.

Reply:
Again this is redundant.